examples/.eslintrc.cjs

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	rules: {
+		/**
+		 * Allow Fluid Framework examples to import from unstable and legacy APIs.
+		 * https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-internal-modules.md
+		 */
+		"import/no-internal-modules": [
+			"error",
+			{
+				allow: [
```

Comment:
Nit: if we exported this array as a separate export, then configs that need to extend this list could do so explicitly, rather than re-declaring the necessary components.

Reply:
I like that idea. Need to find the proper CJS syntax... (it always confuses me a little)

Reply:
@Josmithr, I tried looking this up, but I don't know how to do it.

Reply:
@Josmithr, I tried looking this up, but I don't know how to do it.

Reply:
I think you should be able to just add an entry to `module.exports`. Since this isn't typed, ESLint should just ignore properties it doesn't understand. E.g.:

```js
const internalModulesAllow = [...];

module.exports = {
	rules: {...},
	internalModulesAllow,
}
```

Then the consuming package-level configs that care about that rule can just import the variable as normal:

```js
const { internalModulesAllow } = require(...);
```

Reply:
Ideally, we should be moving to ESLint 9 soon, given that ESLint 8 has now been out of support for a few months... ESLint 9 has ESM support (and TS support forthecoming). That would clean a lot of this up, IMO. But it's going to be a big project...

Reply:
Ah, so just let it pollute the lint config. I think we let it wait until there is a use case.

Reply:
I tried this as I found a use, but alas we get a "no" from tooling:
```text
@fluid-example/webflow: Error: ESLint configuration in .eslintrc.cjs » ../../.eslintrc.cjs is invalid:
@fluid-example/webflow:         - Unexpected top-level property "importInternalModulesAllowed".
@fluid-example/webflow: 
@fluid-example/webflow: Referenced from: /home/jasonha/ff/FluidFramework/examples/data-objects/webflow/.eslintrc.cjs
```
We can work around though it takes mucking with all extends (AFAIK).

Reply:
I looked at the code some more and found more /internal uses in places with missing lint coverage on test code and alterations preventing detection. Fixed all that and implemented a way to share the allow array.